.env.template

```diff
@@ -50,4 +50,14 @@ AZURE_LANGUAGE_SERVICE_KEY=
 # Maximum number of instances in the compute cluster
 # AML_COMPUTE_INSTANCES_NUMBER=
 # Azure Key Vault endpoint
-# AZURE_KEY_VAULT_ENDPOINT=
\ No newline at end of file
+# AZURE_KEY_VAULT_ENDPOINT=
+
+#### Virtual Network
+# Required when deploying the application in network isolation
+# Example: VIRTUAL_NETWORK_ADDRESS_SPACE=10.0.0.0/16
+# Example: PROXY_SERVER_SUBNET_ADDRESS_SPACE=10.0.1.0/24
+# Example: AZURE_RESOURCES_SUBNET_ADDRESS_SPACE=10.0.2.0/24
+# Replace these example values with your actual virtual network configuration
+VIRTUAL_NETWORK_ADDRESS_SPACE=
```

Comment:
Should these be commented out, rather than set as empty strings?
(Generally, the optional things are commented out)

Reply:
In fact, where do these env vars get used?